eclcli/common/utils.py

```diff
@@ -347,6 +347,31 @@ def validate_ipv4(text):
         raise exceptions.CommandError(msg % text)
     return text
 
+def validate_int_range(val, attr_name, min_value=None, max_value=None):
+    try:
+        if not isinstance(val, int):
+            int_val = int(val, 0)
+        else:
+            int_val = val
+        if ((min_value is None or min_value <= int_val) and
+                (max_value is None or int_val <= max_value)):
+            return
+    except (ValueError, TypeError):
+        pass
+
+    if min_value is not None and max_value is not None:
```

Comment:
I think you handle error messages too much. I prefer only one message gets displayed when the argument is invalid. Please reconsider if you can make this function concise.